siac/main.go

```diff
@@ -262,6 +262,9 @@ func main() {
 		renterContractsCmd, renterFilesListCmd, renterFilesRenameCmd,
 		renterFilesUploadCmd, renterUploadsCmd, renterExportCmd,
 		renterPricesCmd)
+
+	renterContractsCmd.AddCommand(renterContractDetailsCmd)
```

Comment:
oh nvm this is siac, the cmd name follows the actual command sequence you type into siac.